### PR TITLE
Skip orka-deploy and run remaining orka-tasks as privileged

### PR DESCRIPTION
**Pull request by vinamra28**
Signed-off-by: vinamra28 <vinjain@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

**Review by vdemeester — APPROVED** [review]

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -118,6 +122,8 @@ function test_non_privileged {
         btest=$(basename $(dirname $(dirname $runtest)))
         in_array ${btest} ${SKIP_TESTS} && { echo "Skipping: ${btest}"; continue ;}
         in_array ${btest} ${PRIVILEGED_TESTS} && continue # We did them previously
+        in_array ${btest} ${ORKA_TASKS} && \
+        oc adm policy add-scc-to-user privileged system:serviceaccount:${tns}:orka-svc || true
```
Comment (piyush-garg) [inline openshift/e2e-tests-openshift.sh]: Where `${tns}` is defined?

File: openshift/e2e-tests-openshift.sh [hunk 2]
```diff
@@ -72,13 +76,18 @@ function test_privileged {
     # Run the privileged tests
     for runtest in $@;do
         btest=$(basename $(dirname $(dirname $runtest)))
-        in_array ${btest} ${SKIP_TESTS} && { echo "Skipping: ${btest}"; continue ;}
+        in_array ${runtest} ${SKIP_TESTS} && { echo "Skipping: ${runtest}"; continue ;}
```
Comment (piyush-garg) [inline openshift/e2e-tests-openshift.sh:79]: This should not be `runtest`

File: openshift/e2e-tests-openshift.sh [hunk 3]
```diff
@@ -72,13 +76,18 @@ function test_privileged {
     # Run the privileged tests
     for runtest in $@;do
         btest=$(basename $(dirname $(dirname $runtest)))
```
Comment (piyush-garg) [inline openshift/e2e-tests-openshift.sh]: This can be removed then

**Review by piyush-garg — APPROVED** [review]

File: openshift/e2e-tests-openshift.sh [hunk 4]
```diff
@@ -13,10 +13,14 @@ MAX_NUMBERS_OF_PARALLEL_TASKS=4
...
-PRIVILEGED_TESTS="buildah buildpacks buildpacks-phases jib-gradle kaniko kythe-go s2i"
+PRIVILEGED_TESTS="buildah buildpacks buildpacks-phases jib-gradle kaniko kythe-go orka-init orka-teardown s2i"
 
 # Skip those tests when they really can't work in OpenShift
-SKIP_TESTS="docker-build orka-full"
+SKIP_TESTS="docker-build orka-full orka-deploy"
+
+# Orka Tasks which can be tested as privileged but existing SA needs to be used
+# to give privileged access
+ORKA_TASKS="orka-init orka-teardown"
```
Comment (chmouel) [inline openshift/e2e-tests-openshift.sh:23]: I don't like much these kinds of specifics tasks, to be honest i would not be against just disabling it, (it's not like it's something critical for openshift pipeline product)

**Pull request merged**